src/frontend/src/views/Data/components/FilterPanelToggle/style.ts

```diff
@@ -1,13 +1,14 @@
+import { css } from "@emotion/css";
 import styled from "@emotion/styled";
-import { getColors, getSpacings } from "czifui";
-import IconFilters from "src/common/icons/IconFilters.svg";
+import { fontHeaderXxxs, getColors, getCorners, getSpacings } from "czifui";
 
 export const StyledDiv = styled.div`
   display: flex;
   align-items: center;
-  justify-content: center;
+  align-content: center;
```

Comment:
do we need both align-items and align-content?

Reply:
yes, one centers vertically and one centers horizontally

Reply:
oops, i must have accidentally added the extra one here when I was trying ot update the badge